docs/how-to-guides/adding-support-for-a-new-online-store.md

```diff
@@ -263,23 +292,82 @@ online_store: feast_custom_online_store.mysql.MySQLOnlineStore
 
 ## 4. Testing the OnlineStore class
 
+### Integrating with the integration test suite and unit test suite.
+
 Even if you have created the `OnlineStore` class in a separate repo, you can still test your implementation against the Feast test suite, as long as you have Feast as a submodule in your repo. In the Feast submodule, we can run all the unit tests with:
 
 ```
-make test
+make test-python
 ```
 
-The universal tests, which are integration tests specifically intended to test offline and online stores, can be run with:
+This should run the unit tests and the unit tests should all pass. Please add unit tests for your online store that test out basic functionality of reading and writing to and from the online store. This should just be class level functionality that ensures that the methods you implemented for the OnlineStore work as expected. In order to be approved to merge into Feast, these unit tests should all pass and demonstrate that the DataSource works as intended.
```

Comment:
Same change and questions as the offline store.

Reply:
Removed this section entirely. Will basically point to universal tests only to validate the datasource